SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1557,22 +1557,24 @@ FRPCErrorInfo USpatialReceiver::ApplyRPC(const FPendingRPCParams& Params)
 
 void USpatialReceiver::OnReserveEntityIdsResponse(const Worker_ReserveEntityIdsResponseOp& Op)
 {
-	if (Op.status_code == WORKER_STATUS_CODE_SUCCESS)
+	if (Op.status_code != WORKER_STATUS_CODE_SUCCESS)
```

Comment:
I'd suggest not logging the status here at all. You have the log later that it succeeded/failed to extract the delegate, and there's logging inside the delegate method. 

If you do want to keep this, I think the 1562 log should be a warning, since we don't know if it's just a timeout.

Reply:
+1 to both of these. Especially error -> warning even if it isn't a timeout as we know we can retry. (unless we want to log spatialos internal errors differently).
Unless we want make it so that delegates only get their callbacks when the request is successful in which case we could log here and no in the entity pool.

(Medium-short term there is also a plan for being able to send commands that auto retry within the gdk if we want to tidy this kind of thing up generally).

Reply:
I went with changing it to a warning, at least for the time being, for 2 reasons:

- Relative pattern consistency with other code in this file,
- Protection against changes to EntityPool (accidentally or intentionally) removing the logs.